projects/step-frontend/src/lib/modules/timeseries/performance-view/filter-bar/filter-bar.component.scss

```diff
@@ -11,22 +11,29 @@ step-ts-filter-bar {
     font-size: 12px;
   }
 
+  .settings-section {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
   .warning-mark {
-    position: absolute;
-    left: 0;
-    top: 0;
     background-color: var.$yellow-10;
     color: var.$yellow-600;
-    padding: 4px 34px;
     font-size: 11px;
-    border: 1px solid var.$yellow-20;
+    padding: 4px 10px;
+    /* width: 100%; */
+    margin: 0 20px;
+    display: flex;
+    margin-top: 12px !important;
   }
 
   .filters {
     display: flex;
     gap: 12px;
     padding: 12px;
     align-items: center;
+    height: 68px;
```

Comment:
We've decided to use `rem` instead of `px` for styling non material parts. Is it possible to replace it in this style sheet?